src/pages/SingleVideoPage/RelatedVideos.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from "react";
+import Loader from "../../components/Loader";
+import NotAvailable from "../../components/NotAvailable";
+import VideoCard from "../../components/VideoCard.jsx";
+import { useVideos } from "../../context/VideosContext";
+
+const RelatedVideos = ({ category, videoId }) => {
+  const { filterVideos, videos } = useVideos();
+  useEffect(async () => {
+    await filterVideos({ category });
+  }, [category]);
+  return (
+    <div className="mt-3">
+      <h3>Related Videos</h3>
+      <button className={`btn font-bold  btn-primary tag mt-2`}>
```

Comment:
What is this button for? If there is no click action associated with it, I'd recommend you use some other tag